assets/css/nav/_top_nav.scss

```diff
@@ -52,3 +53,17 @@
     color: $color-eggplant-800;
   }
 }
+
+.c-top-nav__user-icon {
+  background-color: #6d39ac;
+  color: #ffffff;
+  height: 24px;
+  width: 24px;
+  font-size: 15px;
+  line-height: 24px;
+  border-radius: 12px;
+}
```

Comment:
**issue:** I think this is dead code now that the real button / avatar components are swapped in.

Reply:
💀 https://github.com/mbta/skate/pull/2385/commits/4378d83dc934e15d19f31f2975296cc3472a5668 💀